### PR TITLE
12 school evaluation

### PR DESCRIPTION
**Pull request by AnuKPatel**
Hello,

Migration of **school_evaluation** module is done.
Please review and merge the code.

### CODE REVIEW

File: school_evaluation/views/school_evaluation_view.xml [hunk 1]
```diff
@@ -98,25 +101,22 @@
                              <button name="get_record" type='object' class="oe_inline oe_stat_button"
                                 attrs="{'invisible' : [('state', '!=', 'draft')]}"
                                 groups="school.group_school_administration,school.group_school_teacher">
-                            <img src="/school_evaluation/static/description/index1.png" style="height:25px;width:50px;float:left;"/>
+                            <img src="/school_evaluation/static/description/index1.png" alt="Logo"
+                                style="height:25px;width:50px;float:left;"/>
                             <lable style="">Get Question</lable>
```
Comment (dhirendrasinh-Serpent) [inline school_evaluation/views/school_evaluation_view.xml:106]: No need style attribute.

File: school_evaluation/views/school_evaluation_view.xml [hunk 2]
```diff
@@ -158,25 +158,22 @@
                              <button name="get_record" type='object' class="oe_inline oe_stat_button"
                                 attrs="{'invisible' : [('state', '!=', 'draft')]}"
                                 groups="school.group_school_administration">
-                            <img src="/school_evaluation/static/description/index1.png" style="height:25px;width:50px;float:left;"/>
+                            <img src="/school_evaluation/static/description/index1.png" alt="Logo"
+                                style="height:25px;width:50px;float:left;"/>
                             <lable style="">Get Question</lable>
```
Comment (dhirendrasinh-Serpent) [inline school_evaluation/views/school_evaluation_view.xml:163]: No need style attribute.

**Review by dhirendrasinh-Serpent — COMMENTED** [review]
Improve code

File: school_evaluation/views/school_evaluation_view.xml (hunk 1 above)
Comment (AnuKPatel) [reply school_evaluation/views/school_evaluation_view.xml:106]: It's needed.

**Comment by AnuKPatel** [comment]
Removed the icon attribute from filter tag.

**Pull request merged**